src/EluvioLive.js

```diff
@@ -2928,28 +2928,80 @@ class EluvioLive {
   /**
    * Generate tickets (NTP) as a tenant.
    *
+   * Tickets may be bound to emails - in this case, the list of emails has to be supplied
+   * as an argument, and the 'quantity' is no longer allowed.
+   * Optionally creates embed URLs for each ticket code generated, based on a template URL
+   * that contains all required embed URL parameters except for the ticket and optionally email.
+   *
    * @namedParams
    * @param {string} tenant - The Tenant ID
    * @param {string} otp - The OTP ID
-   * @param {integer} otpClass - The OTP class (4 or 5)
+   * @param {integer} otpClass - The OTP class (4 or 5) (default 5)
    * @param {integer} quantity - Number of tickets to generate
+   * @param {string} emails - File containing one email per line
+   * @param {strnig} embedUrlBase - Base embed URL, used to generate embed URLs for each ticket
```

Comment:
typo here

Reply:
I'm not seeing it - what is it? (I know the English is a little contrived to keep the lines short ...)

Reply:
Oh sorry - line 2942 `{strnig}` instead of `{string}`

Reply:
@elv-sam meant string is misspelled {strnig}

Reply:
@elv-sam meant string is misspelled {strnig}

Reply:
Fixed!